scylla/src/transport/cluster.rs

```diff
@@ -37,11 +37,11 @@ pub struct Datacenter {
 
 #[derive(Clone)]
 pub struct ClusterData {
```

Comment:
Although `ClusterData` is `pub` in this module, the module `cluster` itself is not declared public, therefore the `ClusterData` struct is not accessible from outside of the `scylla` crate. Please make the `cluster` module public.

Additionally:
- Please go over all public methods of `ClusterData` which were not added in this PR and make them `pub(crate)`.
- Make the `Cluster` struct `pub(crate)`.

Reply:
Wouldn't it be less invasive to reexport `ClusterData` with `pub use` in `transport/mod.rs` and keep `cluster` module private? Marking many things as `pub(crate)` is a bit tedious.

```rust
pub use cluster::ClusterData;
```

Reply:
I agree, that's a better idea.

Reply:
Is it ok now?

Reply:
The `ring_range`, `wait_until_all_pools_are_initialized` and `new` methods are still visible. The documentation generated by `cargo doc` shows this:

![image](https://user-images.githubusercontent.com/4950936/144576470-6ff1d54a-deeb-42f4-a529-92ab6d93919f.png)

Those methods weren't exposed to the users before, they are unrelated to the functionality introduced by this PR so we shouldn't expose them in this PR. Changing their visibility to `pub(crate)` should hide them.

To be clear,`get_keyspace_info` is visible and that's OK.